reporter/src/main/kotlin/reporters/NoticeByPackageReporter.kt

```diff
@@ -235,10 +234,9 @@ class NoticeByPackageProcessor(input: ReporterInput) : AbstractNoticeReporter.No
         val licenseFiles = mutableMapOf<String, LicenseFindingsMap>()
 
         archiveDir.walk().forEach { file ->
-            val relativePath = archiveDir.toPath().relativize(file.toPath())
-            val relativePathString = relativePath.toString()
-            if (matcher.matches(relativePathString)) {
-                licenseFiles[relativePathString] =
+            val relativePath = archiveDir.toPath().relativize(file.toPath()).toString()
```

Comment:
This could simply become `val relativePath = file.toRelativeString(archiveDir)`.

Reply:
Done.